apis/datasciencecluster/v1/datasciencecluster_types.go

```diff
@@ -73,8 +73,42 @@ type Components struct {
 
 // ComponentsStatus defines the custom status of DataScienceCluster components.
 type ComponentsStatus struct {
-	// ModelRegistry component status
-	ModelRegistry *componentApi.DSCModelRegistryStatus `json:"modelregistry,omitempty"`
+	// Dashboard component status.
+	Dashboard componentApi.DSCDashboardStatus `json:"dashboard,omitempty"`
+
+	// Workbenches component status.
+	Workbenches componentApi.DSCWorkbenchesStatus `json:"workbenches,omitempty"`
+
+	// ModelMeshServing component status.
+	ModelMeshServing componentApi.DSCModelMeshServingStatus `json:"modelmeshserving,omitempty"`
+
+	// DataServicePipeline component status.
+	// Require OpenShift Pipelines Operator to be installed before enable component
+	DataSciencePipelines componentApi.DSCDataSciencePipelinesStatus `json:"datasciencepipelines,omitempty"`
+
+	// Kserve component status.
+	// Require OpenShift Serverless and OpenShift Service Mesh Operators to be installed before enable component
```

Comment:
```suggestion
	// Requires OpenShift Serverless and OpenShift Service Mesh Operators to be installed before enable component
```

I wonder if this is still true in all cases actually...I guess they shouldn't be absolutely necessary for RawDeployment mode? :thinking:

Reply:
This is something I don't know, the comment was there alrady (just copy pasted, and now removed since it does not make any sense on the status subresource)